app/src/renderer/js/preferences.js

```diff
@@ -62,13 +60,14 @@ document.addEventListener('DOMContentLoaded', async () => {
   audioInputDeviceSelector.value = settingsValues.audioInputDeviceId;
 
   const tabs = $j('.prefs-nav > a');
-  tabs.on('click', function (event) {
+
+  tabs.on('click', event => {
     event.preventDefault();
     tabs.removeClass('is-active');
-    $j(this).addClass('is-active');
+    $j(event.currentTarget).addClass('is-active');
```

Comment:
Don't know if there are more instances where you'd need `currentTarget` instead of `target`.

Reply:
Why not always use `currentTarget` to be sure? `currentTarget` is the bound element (The previous `this`), while `target` is what was clicked, which is usually the same, but if there are more element inside the target, it can be different.

Reply:
Yeah, true. Let's do that.